src/msix/CMakeLists.txt

```diff
@@ -43,10 +43,28 @@ if(XML_PARSER MATCHES msxml6)
     add_definitions(-DUSING_MSXML=1)
 endif()
 
-if(WIN32)
-    set(DirectoryObject PAL/FileSystem/Win32/DirectoryObject.cpp)
+if(CRYPTO_LIB MATCHES crypt32)

```

Comment:
I don't except this to happen but we should probably fail here if someone doesn't use our builds scripts and do for example cmake -DLINUX=on -DCRYPTO_LIB=crypt32 ..

Reply:
I didn't see a similar error for XML_PARSER.  I can look into one for both, but ultimately the compile will just fail to find includes quite quickly.

Reply:
I am ok failing at compile time. The erros is going to be pretty obvious for people to figure out.